node-notes-cli/app.js

```diff
@@ -0,0 +1,61 @@
+const notesJSON = require('./data.json');
+const fs = require('fs');
+
+if (process.argv[2] === 'read') {
+  for (const key in notesJSON.notes) {
+    console.log(`${key}: ${notesJSON.notes[key]}`);
+  }
+}
+
+if (process.argv[2] === 'create') {
+  fs.readFile('data.json', 'utf8', (err, data) => {
+    const JSONParse = JSON.parse(data);
+    if (err) throw err;
+    else {
+      const nextId = JSONParse.nextId++;
+      JSONParse.notes[nextId] = process.argv[3];
```

Comment:
I'd suggest making sure there _is_ a `process.argv[3]` before updating the file.